src/napari_imagej/_function.py

```diff
@@ -233,21 +286,65 @@ def _napari_module_param_additions(module_info) -> Dict[str, Tuple[type, Any]]:
         additional_params["display_results_in_new_window"] = (bool, False)
     return additional_params
 
+def _is_non_default(input):
+    if not input.isRequired(): return False
+    if input.getDefaultValue() is not None: return False
+    return True
+
+
+def _sink_optional_inputs(inputs):
+    """
+    Python functions cannot have non-default args after a default arg.
```

Comment:
Let's stick to *either* optional or default as a way to describe args.

I prefer optional.

Reply:
Fixed in e9b7f32